api/src/main/resources/workbench-api.yaml

```diff
@@ -5534,6 +5534,33 @@ definitions:
         format: int64
         description: Timestamp when the user was bypassed for completing login.gov linking
           linking.
+      renewableAccessModules:
+        modules:
+          type: array
+          items:
+            "$ref": "#/definitions/RenewableAccessModuleStatus"
+        compliance:
+          anyModuleHasExpired:
+            type: boolean
```

Comment:
This does duplicate the information available by scanning all modules in the array, but we believe it is useful to include this summary for clients and worth the tradeoff.